frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
```

Comment:
URL: http://localhost:4200/project/default (Click on the move model button on any model and select a target project)
Issue: Title "Move model" is not large / bold
Screenshot:
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/8f8157e7-e1ae-40b4-8596-bdd135791d9c)